docs/website/blog/3.0.0-GA-blog.md

```diff
@@ -0,0 +1,333 @@
+---
+title: odo v3.0.0 GA Released 🎉
+author: Parthvi Vala
+author_url: https://github.com/valaparthvi
+author_image_url: https://github.com/valaparthvi.png
+tags: ["release"]
+slug: odo-v3-release
+---
+
+odo v3 is now GA! 🎉
+
+![](../static/img/logo_small.png)
+<!--truncate-->
+Are you an application developer wanting to work closely with a production like environment? But can't, because you either don't have the time or energy to dive in the steep learning curve involved in running your application on the cluster?
+
+`odo` is here to answer your problems!
+
+
+Over the past couple of months, odo team has been working on creating a better experience with odo.
+
+## So, what's new?
+
+`odo` v3 comes with a new set of commands that replaces the old ones but essentially does the same thing, in a faster and more efficient way.
```

Comment:
I would rather stick to what we have because the blog is written in a way to tell the audience how it is faster and more efficient than the old commands.

Reply:
Both formulations make sense, they are just looking at things from a slightly different angle.